hack/make.ps1

```diff
@@ -0,0 +1,412 @@
+<#
+.NOTES
+    Author:  @jhowardmsft
+
+    Summary: Windows native build script. This is similar to functionality provided
+             by hack\make.sh, but uses native Windows PowerShell semantics. It does
+             not support the full set of options provided by the Linux counterpart.
+             For example:
+             
+             - You can't cross-build Linux docker binaries on Windows
+             - Hashes aren't generated on binaries
+             - 'Releasing' isn't supported.
+             - Integration tests. This is because they currently cannot run inside a container,
+               and require significant external setup. 
+             
+             It does however provided the minimum necessary to support local Windows 
+             development and parts of Windows to Windows CI.
+
+             Usage Examples (run from repo root): 
+                "hack\make.ps1 -Binary" to build the binaries
+                "hack\make.ps1 -Client" to build just the client 64-bit binary
+                "hack\make.ps1 -TestUnit" to run unit tests
+                "hack\make.ps1 -Binary -TestUnit" to build the binaries and run unit tests
+                "hack\make.ps1 -All" to run everything this script knows about
+
+.PARAMETER Client
+     Builds the client binaries.
+
+.PARAMETER Daemon
+     Builds the daemon binary.
+
+.PARAMETER Binary
+     Builds the client binaries and the daemon binary. A convenient shortcut to `make.ps1 -Client -Daemon`.
+
+.PARAMETER Race
+     Use -race in go build and go test.
+
+.PARAMETER Noisy
+     Use -v in go build.
+
+.PARAMETER ForceBuildAll
+     Use -a in go build.
+
+.PARAMETER NoOptimise
```

Comment:
We _do_ use "fake" english 🇺🇸  everywhere, so I agree that it should be changed for consistency.